<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>模式选取</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2d">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>二维成像结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3d">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>三维成像结果</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>791</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>成像结果</string>
       </property>
       <widget class="QWebEngineView" name="webEngineView">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>19</y>
          <width>791</width>
          <height>571</height>
         </rect>
        </property>
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>291</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string>雷达控制</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>271</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>载频(GHz)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>时宽(ms)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QTextEdit" name="textEdit_fc">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_radar_2">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_radar_1">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>加载默认</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QTextEdit" name="textEdit_B">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTextEdit" name="textEdit_prf">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>带宽(GHz)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QTextEdit" name="textEdit_T">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>脉冲重复频率(Hz)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>系统启动</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>20</y>
          <width>271</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_start">
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>导轨扫描与ZYNQ程序运行开始</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>610</height>
        </size>
       </property>
       <property name="title">
        <string>数据保存情况</string>
       </property>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>271</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>60</height>
        </size>
       </property>
       <property name="title">
        <string>数据浏览</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>271</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>图像名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_local">
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_data">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
